### PR TITLE
Create README #6

### PR DESCRIPTION
**Pull request by furesoft**
This solves #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+[![Unit Tests](https://github.com/Backseating-Committee-2k/BackseatSafeSystem2k/actions/workflows/rust.yml/badge.svg)]([https://github.com/Backseating-Committee-2k/Seatbelt/actions/workflows/cmake.yml](https://github.com/Backseating-Committee-2k/BackseatSafeSystem2k/actions/workflows/rust.yml))
+[![Licence: MIT](https://img.shields.io/github/license/Backseating-Committee-2k/BackseatSafeSystem2k)](https://github.com/Backseating-Committee-2k/BackseatSafeSystem2k/blob/main/LICENSE)
+[![Discord Channel](https://img.shields.io/discord/834834066008309800?style=social)](https://discord.gg/WygnW2wZj3)
+[![Twitch live status](https://img.shields.io/twitch/status/coder2k?style=social)](https://twitch.tv/coder2k)
+
+# BackseatSafeSystem2k
+
+This is the official implementation of the platform invented from the twitch stream Coder2k.
```
Comment (mgerhold) [inline README.md]:
```suggestion
This is the official implementation of the platform invented by the Twitch chat of [coder2k](https://twitch.tv/coder2k).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Features
+
+- Textmode
+- Videomode
+- Debugging-Instructions
+
+## How to build
+
+The project uses [cargo]([https://cmake.org/](https://crates.io)) as build system. All dependencies are fetched automatically. Build the
```
Comment (mgerhold) [inline README.md]:
```suggestion
The project uses [cargo](https://crates.io/) as build system. All dependencies are fetched automatically. Build the
```

**Review by mgerhold — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Features
+
+- Textmode
+- Videomode
+- Debugging-Instructions
+
+## How to build
+
+The project uses [cargo]([https://cmake.org/](https://crates.io)) as build system. All dependencies are fetched automatically. All dependencies are fetched automatically. Build the
```
Comment (mgerhold) [inline README.md]:
```suggestion
The project uses [cargo](https://crates.io) as build system. All dependencies are fetched automatically. Build the
```

**Review by mgerhold — CHANGES_REQUESTED** [review]

**Review by furesoft — COMMENTED** [review]
Maybe this time :D

File: README.md [hunk 4]
```diff
@@ -15,8 +15,7 @@ This is the official implementation of the platform invented by the Twitch chat
 
 ## How to build
 
-The project uses [cargo]([https://crates.io](https://crates.io)) as build system. All dependencies are fetched automatically. All dependencies are fetched automatically. Build the
-software like so:
+The project uses [cargo]([https://crates.io](https://crates.io)) as build system. All dependencies are fetched automatically.  Build the software like so:
```
Comment (mgerhold) [inline README.md]:
```suggestion
The project uses [cargo](https://crates.io) as build system. All dependencies are fetched automatically.  Build the software like so:
```

**Review by mgerhold — CHANGES_REQUESTED** [review]

**Review by mgerhold — APPROVED** [review]

**Comment by mgerhold** [comment]
fixes #6

**Pull request merged**